src/screens/docs/components/victory-area/ecology.md

```diff
@@ -1,260 +1,410 @@
 # VictoryArea
 
-VictoryArea renders a dataset as a single area. VictoryArea can be composed with [VictoryChart] to create area charts.
+VictoryArea renders a dataset as a single area. VictoryArea can be composed with [`VictoryChart`] to create area charts.
 
 ```playground
-<VictoryArea
-  data={[
-    {month: "September", profit: 35000, loss: 2000},
-    {month: "October", profit: 44000, loss: 8000},
-    {month: "November", profit: 55000, loss: 5000}
-  ]}
-  x="month"
-  y={(datum) => datum.profit - datum.loss}
-/>
+<VictoryChart
+  theme={VictoryTheme.material}
+  style={{
+    parent: { border: "1px solid #ccc"}
+  }}
+>
+  <VictoryArea
+    style={{
+      data: { fill: "#c43a31" },
+      parent: { border: "1px solid #ccc"}
+    }}
+    data={[
+      { x: 1, y: 2 },
+      { x: 2, y: 3 },
+      { x: 3, y: 5 },
+      { x: 4, y: 4 },
+      { x: 5, y: 7 }
+    ]}
+  />
+</VictoryChart>
 ```
 
 ## Props
 
-### data
+### `animate`
 
-Specify data via the `data` prop. By default, Victory components expect data as an array of objects with `x` and `y` properties. Use the [`x` and `y`] data accessor props to define a custom data format. The `data` prop must be given as an array.
+`VictoryArea` uses the standard `animate` prop. [Read about it here](https://formidable.com/open-source/victory/docs/common-props#animate)
```

Comment:
so just have the example from the common prop in VictoryArea etc, and the docs in common props?